src/docs/product/integrations/gcp-cloud-run/index.mdx

```diff
@@ -0,0 +1,15 @@
+# Google Cloud Run
+
+Cloud Run helps developers save time in building and deploying their applications. Sentry helps save time in resolving production issues by providing detailed debugging capabilities.
+
+### Setup
+
+1. [Create an account](https://sentry.io/signup/) (or [Login](https://sentry.io/auth/login/)) on Sentry
+2. Choose your application platform
+
+    ![Sentry Choose Platform](sentry-choose-platform.png)
+
+3. Follow instructions to instrument your Cloud Run application. Detailed platform docs [here](http://docs.sentry.io/).
+4. Deploy your Cloud Run application with GCP.
+
+That's it! Sentry will now report all issues from your Cloud Run application.
```

Comment:
If you would like, you can use this standard verbiage "Once this is done, all unhandled exceptions are automatically captured by Sentry."

Reply:
I will leave it as it is since we will capture transactions a well.